app/subjects/mention-list.cjsx

```diff
@@ -35,13 +35,14 @@ module.exports = React.createClass
       <div>
         <h2>Mentions:</h2>
         <div>
-          {for mention, i in @state.mentions
+          {@state.mentions.map (mention) ->
             <Comment
-              key={"mention-#{mention.comment.id}-#{i}"}
+              key={"mention-#{mention.comment.id}"}
```

Comment:
It needed the index.  When there are multiple mentions per comment they will have the same key.